### PR TITLE
[Workflow] Add `getEnabledTransition()` method

### LINKED ISSUE #19234: [Workflow] Add `getEnabledTransition()` method annotation to WorkflowIn…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#52854
| PR author(s) | @alexandre-daubois
| Merged in    | 7.1

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.1 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by ChibyJade**
Fix #19234

### CODE REVIEW

File: workflow.rst [hunk 1]
```diff
@@ -310,6 +310,15 @@ machine type, use ``camelCased workflow name + StateMachine``::
         }
     }
 
+To get all enabled transitions of a Workflow, you can use
```
Comment (ChibyJade) [inline workflow.rst]:
```suggestion
To get the enabled transition of a Workflow, you can use
```

File: workflow.rst [hunk 2]
```diff
@@ -310,6 +310,15 @@ machine type, use ``camelCased workflow name + StateMachine``::
...
+:method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransitions`
```
Comment (ChibyJade) [inline workflow.rst]:
```suggestion
:method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition`
```

File: workflow.rst [hunk 3]
```diff
@@ -310,6 +310,15 @@ machine type, use ``camelCased workflow name + StateMachine``::
...
+method.
+
+.. versionadded:: 7.1
+
+    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransitions` method
```
Comment (ChibyJade) [inline workflow.rst]:
```suggestion
    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition` method
```

File: workflow.rst [hunk 4]
```diff
@@ -310,6 +310,15 @@ machine type, use ``camelCased workflow name + StateMachine``::
...
     }
 
+To get the enabled transition of a Workflow, you can use
+:method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition`
+method.
+
+.. versionadded:: 7.1
+
+    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition` method
+    was introduced in Symfony 7.1.
```
Comment (OskarStark) [inline workflow.rst]:
```suggestion
    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition`
    method was introduced in Symfony 7.1.
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @ChibyJade.

**Pull request merged**